packages/core/src/test/techdebt.test.ts

```diff
@@ -17,17 +17,6 @@ describe('tech debt', function () {
         assert.ok(now <= cutoffDate, msg)
     }
 
-    it('vscode minimum version', async function () {
-        const minVscode = env.getMinVscodeVersion()
-        assert.ok(semver.lt(minVscode, '1.84.0'))
-
-        // see https://github.com/microsoft/vscode/issues/173861
-        assert.ok(
-            semver.lt(minVscode, '1.93.0'),
```

Comment:
unless we have some other way of detecting that 173861 is fixed, we should probably keep this test around as a reminder to check 173861. We can set the version here to 1.110, then "snooze" it to 1.130, etc.